src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -3435,6 +3435,20 @@ get_client (void)
    return test_framework_new_default_client ();
 }
 
+/* Returns a test client without version API options configured. */
+static mongoc_client_t *
+get_client_for_version_api_example (void) {
+   mongoc_client_t *client;
+   mongoc_uri_t *uri;
+
+   uri = test_framework_get_uri ();
+   client = mongoc_client_new_from_uri (uri);
+   ASSERT (client);
```

Comment:
I'm assuming that since this is test code that we expect to just terminate, there's no need to release the "client" resource?

Reply:
Callers of `get_client_for_version_api_example` are expected to call `mongoc_client_destroy` to release the `client`.